packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -103,6 +103,14 @@ abstract class VideoPlayerPlatform extends PlatformInterface {
   Future<void> setMixWithOthers(bool mixWithOthers) {
     throw UnimplementedError('setMixWithOthers() has not been implemented.');
   }
+
+  Future<void> setupAudioSession(AudioSession config) {
+    throw UnimplementedError('setupAudioSession() has not been implemented.');
```

Comment:
This should be a no-op, not a throw, otherwise init will throw for all existing implementations.